workload-metrics/manifests/workload-metrics-hpa.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: autoscaling/v2beta2
```

Comment:
All YAML samples need licenses and region tags. 
go/code-snippets-101 

Example: 
https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/master/hello-app/manifests/helloweb-deployment.yaml